public/index.html

```diff
@@ -8,6 +8,7 @@
   <meta name="theme-color" content="#000000" />
   <meta name="description" content="Web site created using create-react-app" />
   <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+  <script src="https://kit.fontawesome.com/7f2dbdaff1.js" crossorigin="anonymous"></script>
```

Comment:
Can you please take a look how you can use fontawesome in React? https://fontawesome.com/v5/docs/web/use-with/react
Also would like to mention that there are material icons https://mui.com/material-ui/material-icons/.

Reply:
I have used material icons, but not sure if i describe the TS interface right..